f/php/bootstrap

```diff
@@ -0,0 +1,144 @@
+#!/opt/bin/php -c/opt/php.ini
+<?php
+use Bref\Bridge\Psr7\RequestFactory;
+use Bref\Http\LambdaResponse;
+use Symfony\Component\Process\Process;
+
+echo 'Cold start' . PHP_EOL;
+
+ini_set('display_errors', '1');
+error_reporting(E_ALL);
+
+$lambdaRuntimeApi = getenv('AWS_LAMBDA_RUNTIME_API');
+
+require __DIR__ . '/vendor/autoload.php';
+
+$server = new Process("php -S localhost:8000 'index.php'");
+$server->setTimeout(null);
+$server->start(function ($type, $output) {
+    if ($type === Process::ERR) {
+        echo($output);
+        exit(1);
```

Comment:
Not sure what the best way to handle errors is. Does the instance terminate when the bootstrap is done?

Reply:
For the benchmark as long as it doesn't impact performances I think it's something we can ignore. If we do build that solution for real then we can make it extra robust and think about this.

> Does the instance terminate when the bootstrap is done?

I *think* the lambda is stopped if the bootstrap process stops. So yeah you probably don't want to `exit()` here. But again, not a big deal here as long as it works.